lib/diagnostic.rb

```diff
@@ -136,16 +159,23 @@ def punch
     @moves << 'punch'
     @damage += 5
     @damage *= multiplier
+    self
   end
 
   def move(direction)
     @moves << "move #{direction}"
+    self
   end
 
   def kick
     @moves << 'kick'
     @damage += 10
     @damage *= multiplier
+    self
+  end
+
+  def.get_possible_moves
+    'kick, move, punch'
   end
 
   private
```

Comment:
You're  so close to getting it to run! 
Just need to alter `get_possible_moves`